debug/src/debug.js

```diff
@@ -221,6 +222,22 @@ export function initDebug() {
 							serializeVNode(vnode) +
 							`\n\n${getOwnerStack(internal)}`
 					);
+				}else if (
+					!isCompatNode &&
+					key === 'style' &&
+					vnode.props[key] !== null &&
+					typeof vnode.props[key] === 'object'
+				) {
+					const style = vnode.props[key];
+					for (let i in style) {
+						if (typeof style[i] === 'number' && !IS_NON_DIMENSIONAL.test(i)) {
+							console.warn(
+								`Inline CSS value is missing a unit like "rem" or "px: "${i}: ${style[i]}"\n` +
```

Comment:
```suggestion
								`Inline CSS value is missing a unit like "rem" or "px": ${i}: ${style[i]}"\n` +
```

Reply:
Though tbh suggesting `rem` here might be misleading - unitless values were being converted to px, never rem.

```suggestion
								`Numeric CSS property value is missing a "px" unit: ${i}: ${style[i]}"\n` +
```